project-fob/Models/Attendee.cs

```diff
@@ -7,17 +8,51 @@ public class Attendee
     {
         [Key]
         public int Id { get; set; }
-        public virtual Meeting Meeting { get; set; }
-        public virtual User User { get; set; }
+        public Meeting Meeting { get; set; }
+        public  User User { get; set; }
 
         public Attendee() 
         {
         }
 
         public Attendee(User user, Meeting meeting)
         {
+            if (user == null)
+            {
+
+            }
```

Comment:
This block of code isn't doing anything.

Reply:
breakpoints test code ... whoops... should've removed it.